doc/reference/configuration/cfg_storage.rst

```diff
@@ -61,6 +63,26 @@
     | Environment variable: TT_MEMTX_MIN_TUPLE_SIZE
     | Dynamic: no
 
+.. _cfg_storage-memtx_allocator:
+
+.. confval:: memtx_allocator
+
+    Since version :doc:`2.10.0 </release/2.10.0>`.
+    Specifies the allocator used for memtx tuples.
+    The possible values are ``system``  and ``small``:
```

Comment:
Nit: maybe just "Possible values:" for conciseness?

Reply:
Thanks, I use the full sentence here to make some kind of intro for the subsequent descriptions after the colon.